### PR TITLE
Progress-Module 1

### PR DESCRIPTION
**Pull request by Hsuwaddyhtat22**
.Added HTML
.Added CSS
.Added styling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,15 +154,12 @@ Example:
...
-- [ ] **[more styling with Css]**
+- **[Header]**
+- **[Footer]**
+- **[more styling with Css]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
 
 ## 🤝 Contributing <a name="contributing"></a>
```
Comment (Eternalgratis) [inline README.md:165]: - Some of your social media links in your README are broken. Kindly attach the correct links .
- All the tasks for your project should be done in the feature branch that was created from your main but you currently have most of your work in the main. Kindly update your feature branch so your tasks can reflect there

**Review by Eternalgratis — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️
Hi @Hsuwaddyhtat22 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

almost-there

 ## Highlights
✅ Descriptive pull request message
✅ linter error checks validated

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Hsuwaddyhtat22** [comment]
@Eternalgratis 
When I submit new what pull request shoul submit.pull or pull 2

**Comment by Hsuwaddyhtat22** [comment]
https://github.com/Hsuwaddyhtat22/Hello-Microverse/pull/2
@Eternalgratis Eternalgratis
plz recommend my pull request . What do I need to do?

**Review by AbdusSattar-70 — COMMENTED** [review]
### STATUS: INVALID 
Hi @Hsuwaddyhtat22, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request


I regret to inform you that I am unable to review your submitted project at this time. The social media link and live demo link you have provided appear to be broken or inaccessible. I kindly request that you ensure you are using your own functional social media link in order to facilitate the review process. Your attention to this matter would be greatly appreciated. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Bantealem — APPROVED** [review]
Hi @Hsuwaddyhtat22  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [x] Correct GitHub flow.

## Optional suggestions
- [ ] Although the previous code reviewers didn't ask you to make changes I strongly recommend setting up the `Linters` configuration for this project. you can [find it here](https://github.com/microverseinc/linters-config)
- [ ] Please make sure you only write the correct tech stack for this project in your readme file. Kindly. It should be only `HTML` and `CSS` for this project.
Screenshot from 2023-06-01 11-16-21


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**